CMSgov/design-system
# [WNMGDS-1816] Redirect child doc sites

## Summary

- Removing child documentation site content
- Adding pages that will alert the user that the doc site has moved and redirect them to the unified doc site

## How to test

1. Checkout this branch
2. Open the `index.html` file in the medicare directory. Confirm that a redirect notice is available and the page redirects
3. Open the `index.html` file in the healthcare directory. Confirm that a redirect notice is available and the page redirects

### If this is a change to code:


### If this is a change to documentation:
